### PR TITLE
Feature/timer functionality

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [x] JSX
- [ ] HTML
- [x] JavaScript
- [ ] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
ETA data flow / timer component

## Please give a short description of the changes that were made below

- Feature:
- Created a `handleEtaChange` function in App.js, which drills down to Form.js, where it is used in a useEffect() function to update the ETA prop (which is then lifted to state) whenever the mutation succeeds.  
- ETA prop is then passed from App.js to the TripView component.  We can't connect the ETA to the display until we implement Router, but if you look at the React Component tab in dev tools, TripView is successfully receiving that ETA value.
- Added the two other timers, and a `renderTime` function to display the correct unit and amount of time in each (hours, minutes, seconds).
- Built out a `reduceTime` function to take in our ETA in minutes and reduce it into hours and minutes, when applicable.
- Added a CreateTrip mutation alias to our Cypress testing - no actual tests using this mutation, though.
- Next up:  figure out how to synchronize the timers, trigger the seconds timer on each minute change, and eventually how to trigger our SMS messaging function when countdown ends.

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
-    <div className="App">
-      {/* <LoginView /> */}
-      {/* <MainView /> */}
-      <TripView />
-    </div>
-  );
+
+  const [eta, setETA] = useState('');
+
+  const handleEtaChange = (time) => {
```
Comment (carolineshowscode) [inline src/Components/App/App.js:10]: nice job figuring out how to lift state

File: src/Components/Form/Form.js [hunk 2]
```diff
@@ -36,9 +36,11 @@ function Form({contacts}) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  // if (loading) return 'Loading...';
-  // if (error) return `Error! ${error.message}`;
-  // if (data) return `${console.log(data.createTrip.trip.eta)}`;
+  useEffect(() => {
+    if (data) {
+      handleEtaChange(data.createTrip.trip.eta);
```
Comment (bhohnco) [inline src/Components/Form/Form.js:41]: Nice work here! This will be solid to use.

File: src/Components/TripView/TripView.js [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
+  }, []);
+
+  const reduceTime = () => {
+    if (mockETA > 60) {
+      setHours(Math.floor(mockETA / 60 ));
+      setMinutes(mockETA % 60);
+    }
+  }
+
+  const renderTime = (unit, time) => {
```
Comment (bhohnco) [inline src/Components/TripView/TripView.js:24]: Looking forward to checking this out. Nice work!

**Review by bhohnco — APPROVED** [review]

**Review by carolineshowscode — APPROVED** [review]
As we discussed in today's meeting. In addition to the timer we need to add the following happy/sad paths:
- timer is up -> user requests more time
- timer is up (timer in background starts) and user doesn't end trip or add more time -> emergency text sent to contact
- time is up & user ends trip

**Pull request merged**